# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1285748757 14400
#      Wed Sep 29 04:25:57 2010 -0400
# Node ID b099acf06b556e807caea8a0d6e621da3d35277c
# Parent  2204120e38d266325115c09223cd4a4568dfd9b5
make_int.cc: don't pass arg to mlock

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2010-09-29  John W. Eaton  <jwe@octave.org>
+
+	* examples/make_int.cc (Fmake_int): Don't pass arg to mlock.
+
 2010-09-29  John W. Eaton  <jwe@octave.org>
 
 	* examples/structdemo.cc (Fstructdemo): Use octave_scalar_map
 	instead of Octave_map.
 
 2010-09-29  John W. Eaton  <jwe@octave.org>
 
 	* configure.ac: Style fixes.
diff --git a/examples/make_int.cc b/examples/make_int.cc
--- a/examples/make_int.cc
+++ b/examples/make_int.cc
@@ -235,17 +235,17 @@ DEFUN_DLD (make_int, args, ,
 \n\
 Creates an integer variable from VAL.")
 {
   static bool type_loaded = false;
 
   if (! type_loaded)
     {
       octave_integer::register_type ();
-      mlock ("make_int");
+      mlock ();
 
       octave_stdout << "installing integer type at type-id = "
            << octave_integer::static_type_id () << "\n";
 
       INSTALL_UNOP (op_not, octave_integer, gnot);
       INSTALL_UNOP (op_uminus, octave_integer, uminus);
       INSTALL_UNOP (op_transpose, octave_integer, transpose);
       INSTALL_UNOP (op_hermitian, octave_integer, hermitian);
